### ISSUE TITLE
I.L.: Launch Income Limits

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#14294]
### User story

**AS A** Veteran
**I WANT** VA to launch the IL wizard
**SO THAT** I can get access to the information I need.

#### Prep:
- [ ] CAIA publishes the "Learn more about income limits and deductions" R&S page – IL links out to it, so it needs to be live before IL goes
  - UPDATE: confirming that Megan will publish R&S page Tues morning ([slack](https://dsva.slack.com/archives/C52CL1PKQ/p1693258314284069?thread_ts=1693250684.582959&cid=C52CL1PKQ))
- [x] PW updates the link to that page on the Results screen to the correct url (and this needs to merge to prod pre-launch)

#### Day-of – coordinate with Megan Zehnder for CAIA stuff
- [x] DaveC provides greenlight
- [x] confirm it's _**not**_ Friday
- [x] PW flip IL on in prod using registry - _this will go at daily code deploy_
- [x] CAIA publishes inbound links to IL (by updating multiple Hub pages)

#### Fast-follow
- [x] CAIA updates CTA link on R&S page to point to new app instead of Teamsites link
- PW redirects from old Teamsite IL page to the new app's `/introduction` screen (#14964)
- PW enables Cypress suite when Platform is ready (when Cypress 12 rolls forward) (may need a separate ticket).  Currently skipped in build; hoping that Platform's upgrade to Cypress12 fixes flakiness

#### pre-launch:
- [x] Updated R&S link – "Learn more about income limits and deductions" link on the Results page links to the real page at `[environment]/resources/va-health-care-income-limits`
  - [x] Verified in Staging

#### Launch day
- [x] DaveC greenlit launch
- [x] NOT on a Friday
- [x] The app is live in prod
- [x] AFTER prod verification of app, coordinated with Megan Zehnder/CAIA to release content that points to it ([slack thread](https://dsva.slack.com/archives/C01K37HRUAH/p1692911142284729?thread_ts=1692908852.920739&cid=C01K37HRUAH) discussing PoC)
- [x] PM/PO/CAIA verified in Prod that everything is linked together and working as expected

**Comment by randimays** [comment]
@wesrowe In the word doc, the first sentence on the Results screen is 
> We use your last year’s household income, minus certain deductions...

I didn't see specific wording for this for the Past income limits flow. In the case that the veteran has selected 2018 for the Past flow, would this make more sense?
> We use your 2017 household income, minus certain deductions...

**Comment by wesrowe** [comment]
@randimays, thanks for reading this with a critical eye. Unless we have another version somewhere, I think 

Looking at the full text of the current-year results screen, I think we might want to tweak it a bit more for past-years.

> **Your income limits for [selected year]**
>
>We use your last year’s household income, minus certain deductions, to determine your VA health care eligibility and costs. Select your income range to find out how your income may affect your eligibility and costs. But remember, the only way to know for sure is to apply.  
>
> How to estimate your household income range: 
> 
> - Add up all the money your household earned in [previous year] from jobs and other sources. Your household includes you and your spouse and dependents if you have them. 
> - Then subtract any deductions you had in [previous year]. You can deduct some health care costs you didn’t get paid back for, expenses for your own college or vocational training, and any funeral or burial expenses for a spouse or dependent child. 

Things I noticed:
- I agree with Randi's observation that we should adjust "last year's household income" to something like "previous year's"
- It doesn't make sense to me to urge a past-year viewer to apply. We think they already have healthcare to even be interested in past years
- present tense feels weird In "Your household includes you and your spouse and dependents if you have them." It's in the past, it should be clearer that their dependents that year is what matters. 

@thejordanwood, do you remember seeing any versions of the previous-year copy docs that had adjusted the language for the past year flow? If not, we seem to need to do a cycle with Danielle.

**Comment by thejordanwood** [comment]
@wesrowe This was several copy cycles ago, and now I'm not sure if Danielle wrote this or not.  We had something like this for the first paragraph of the results screen. 

> We use your previous year's income to determine your VA health care eligibility and costs. Select your household's 2020 income range. We'll tell you how income limits may affect you.

**Comment by randimays** [comment]
@wesrowe I just saw a comment that Dave C. had posted on Danielle's Word doc about the `Learn more about income limits and deductions` link on the Results page. He said:

> This currently links to Drupal preview environment. Once finalized and published, will link to va.gov/resources/va-health-care-income-limits (assuming the h1 does not change)

I'll go ahead and put this link in there for now, and we can re-examine and adjust if needed before launch.

**Comment by randimays** [comment]
Update on the Cypress front: The Platform team has [opened a PR](https://github.com/department-of-veterans-affairs/vets-website/pull/25149) for the Cypress 12 upgrade. I tested locally and all our Income Limits tests are passing. I'll get a PR up next week to re-enable them and make sure they make it through CI.

**Comment by wesrowe** [comment]
@jilladams, regarding redirects for I.L. – I just saw that the IA team originally stated in the [IA intake ticket](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/income-limits.md) that we need one redirect. (very bottom of doc). I updated the ACs. 

Should that get its own ticket? 

I'll ask the IA team about timing, like should we try to time the redirect with the launch, or can it come after.

**Comment by jilladams** [comment]
I'm agnostic on whether it gets its own ticket. Either way (here or new ticket), we just need to make sure to call out required timing and the need for an OOB deploy if a default Wed deploy won't work.

**Comment by wesrowe** [comment]
@DanielleThierryUSDSVA @rlhecht, we are getting very close to launching Income Limits! There is some publishing that needs to happen around launch, so I've laid out the steps as I know them in the body of this "launch" ticket.

First thing, we need to add the real url for the link on the Results screen, "Learn more about income limits and deductions." I found this URL in the cms: `https://prod.cms.va.gov/resources/va-health-care-income-limits`. Is it correct?

If it's correct, we'll add it to the code now as it exists in Staging only, so that when we make the app visible in Prod the link will be there. Lmk.

**Comment by randimays** [comment]
Re: Cypress 12 / enabling IL e2e tests

Platform upgraded us to Cypress 12 today. [Here's the PR](https://github.com/department-of-veterans-affairs/vets-website/pull/25390) for enabling the tests in the CI. I didn't have any issues with running them locally. But we'll have to monitor over the next few days to see if they get flaky in the pipeline, just to be sure.

**Comment by randimays** [comment]
**Edit** on the Cypress 12 upgrade above. The upgrade caused some linting errors across vets-website that other teams need to address before we can stay on Cypress 12. I rolled back the changes to enable the IL Cypress suite for now.

**Comment by jilladams** [comment]
Launch thread with CAIA - https://dsva.slack.com/archives/C01K37HRUAH/p1693250883860039?thread_ts=1692908852.920739&cid=C01K37HRUAH

Megan Zehnder to publish CAIA's page tomorrow morning (Tues 8/29). After she publishes, we can make the Results screen link change. We can prep that PR first thing tomorrow.

**Comment by jilladams** [comment]
App is live! https://www.va.gov/health-care/income-limits/

Gave Content team the greenlight to update inbound links: https://dsva.slack.com/archives/C52CL1PKQ/p1693340807988469?thread_ts=1693250684.582959&cid=C52CL1PKQ

**Comment by jilladams** [comment]
Content team links are updated, CAIA / Wes have verified, closing. 

Redirect is queued to fast follow this week.
